calculator.py

```diff
@@ -8,7 +8,7 @@ def multiply(x, y):
     pass
 
 def divide(x, y):
-    pass
+    return x/y
```

Comment:
@Guichp Please review this code, is not up to the expectations. You should've considered the division by 0.
:smile:

Reply:
🤫